oaknational/Oak-Web-Application
# feat(bio-cards): add bio card and bio card list components

## Description

- creates `AvatarImage` component
- extends `SocialButtons` to take profiles as props
- adds test for `SocialButtons`
- creates the `BioCardList` component
- fixes icon button sizes

## Issue(s)

progresses #475 

## How to test

1. Go to {srorybook}
2. Click on BioCardList
3. You should see cards like screenshot below

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/194370048-50f0ea5e-e3f9-437a-b078-d569d51c4a62.png)

## Checklist
